service/src/device_method.ts

```diff
@@ -52,12 +52,11 @@ export class DeviceMethod {
    * @description       Invokes the method on the specified device with the specified payload.
    * @param {String}    deviceId    Identifier of the device on which the method will run.
    * @param {Function}  done        The function to call when the operation is
-   *                                complete. `done` will be called with three
-   *                                arguments: an Error object (can be null), a
-   *                                {@link module:azure-iothub.Twin|Twin}
-   *                                object representing the created device
-   *                                identity, and a transport-specific response
-   *                                object useful for logging or debugging.
+   *                                complete. done` will be called with three
+   *                                arguments: an Error object (can be null), the
+   *                                body of the response, and a transport-specific
+   *                                response object useful for logging or
+   *                                debugging.
```

Comment:
yeah i'm on the fence for this. since we advertise it, removing it is a breaking change technically, but at the same time it'd be bad practice for users to use it. emphasize "not for production use" maybe?

---
In reply to: [176866996](https://github.com/Azure/azure-iot-sdk-node/pull/256#discussion_r176866996) [](ancestors = 176866996)